src/components/designer/taro/actions/OpenMintAssetModal.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { WalletOutlined } from '@ant-design/icons';
+import styled from '@emotion/styled';
+import { Button } from 'antd';
+import { usePrefixedTranslation } from 'hooks';
+import { TaroNode } from 'shared/types';
+import { useStoreActions } from 'store';
+
+const Styled = {
+  Button: styled(Button)`
+    width: 50%;
+  `,
+};
+interface Props {
+  node: TaroNode;
+  isContextMenu?: boolean;
+}
+
+const OpenMintAssetModal: React.FC<Props> = ({ node, isContextMenu }) => {
+  const { l } = usePrefixedTranslation('cmps.designer.taro.ActionsTab.MintAsset');
```

Comment:
The key should be the same as the component name.

Reply:
Done